model/task/db.go

```diff
@@ -2866,6 +2867,18 @@ func SetHasAnnotations(taskId string, execution int) error {
 	return errors.Wrapf(err, "marking task '%s' as having annotations", taskId)
 }
 
+// SetNumNextTaskDispatches sets the number of times a host has requested this
+// task and execution as its next task.
+func (t *Task) SetNumNextTaskDispatches() error {
```

Comment:
Can we call this `IncNumNextTaskDispatches` or something since it's doing a +1 now instead of setting the value?